ckanext/switzerland/templates/organization/member_new.html

```diff
@@ -20,19 +21,26 @@
               disabled="True" class="form-control">
             {% else %}
                 <input id="username" type="text" name="username" placeholder="{{ _('Username') }}"
-              value="" class="control-medium" data-module="autocomplete"
-              data-module-source="/api/2/util/user/autocomplete?q=?">
+                value="" class="control-medium" data-module="autocomplete"
+                data-module-source="/api/2/util/user/autocomplete?ignore_self=true&q=?">
             {% endif %}
           </div>
         </div>
       </div>
     </div>
 
-    {% set format_attrs = {'data-module': 'autocomplete'} %}
-    {{ form.select('role', label=_('Role'), options=roles, selected=user_role, error='', attrs=format_attrs) }}
+    {% if user and user.name == c.user and user_role == 'admin' %}
```

Comment:
This block is this way in the CKAN template and I don't know why it's like this, but I don't want to debug it right now. :-/

Reply:
Maaaaaybe add this comment as a TODO or do you have it in the commit-message?